app/auth/create.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react'
+import { Text, TextInput, HelperText, Button } from 'react-native-paper'
+import { useRouter } from 'expo-router'
+import { useDispatch, useSelector } from 'react-redux'
+
+import AuthPage from '@components/AuthPage'
+import { createUser, selectIsLoggedIn } from '@store'
+
+const CreateAccount = () => {
+    const dispatch = useDispatch()
+    const router = useRouter()
+
+    const [fName, setFName] = useState('')
+    const [lName, setLName] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [passwordConfirm, setPasswordConfirm] = useState('')
+    const [badEmail, setBadEmail] = useState(false)
+    const [passwordsMatch, setPasswordsMatch] = useState(true)
+
+    useEffect(() => {
+        const re = /^[\w-\\.]+@crimson.ua.edu$/
+        if (email.length > 0 && !re.test(email)) {
+            setBadEmail(true)
+        } else {
+            setBadEmail(false)
+        }
+    }, [email])
+
+    useEffect(() => {
+        if (passwordConfirm.length > 0 && passwordConfirm == password) {
+            setPasswordsMatch(true)
+        } else {
+            setPasswordsMatch(false)
+        }
+    }, [isLoggedIn])
+
+    const onCreatePress = async () => {
+        //TODO: form validation
+        await dispatch(createUser)
+    }
+
+    const isLoggedIn = useSelector(selectIsLoggedIn)
```

Comment:
This line and the `useEffect` hook that follow it should be automatically handled by the `AuthPage` component.